purchases/src/main/kotlin/com/revenuecat/purchases/PurchasesConfiguration.kt

```diff
@@ -62,6 +65,9 @@ open class PurchasesConfiguration(builder: Builder) {
         @set:JvmSynthetic @get:JvmSynthetic
         internal var dangerousSettings: DangerousSettings = DangerousSettings()
 
+        @set:JvmSynthetic @get:JvmSynthetic
+        internal var pendingPrepaidSubscriptionsEnabled: Boolean = true
```

Comment:
maybe we can default to false for now while the FF is disabled by default?

Reply:
it's default off for BillingClient after all (in that you need to call something explicitly to enable it)

Reply:
I think updating to default to true in the future would be fine and wouldn't require a major since it won't have any API changes

Reply:
Renamed in https://github.com/RevenueCat/purchases-android/pull/1717/commits/67a43a68ca9aad093b14cd717f942a85c4ede392